### PR TITLE
Ws 050 pending sighting

### PR DESCRIPTION
**Pull request by Annie1790**
Hi @Aude11 ,

Can you please have a look on my changes?

### CODE REVIEW

File: src/main/java/com/whalespottingjava/controllers/SightingController.java [hunk 1]
```diff
@@ -65,4 +65,11 @@ public String getAllSightings(Model model) {
   @GetMapping("/sightings/approved")
   @ResponseStatus(HttpStatus.I_AM_A_TEAPOT)
   public void getAllApprovedSightings() {}
+
+  @GetMapping("/sightings/pending")
+  @ResponseStatus(HttpStatus.OK)
+  public String getPendingSightings(Model model) {
+    model.addAttribute("sightings", sightingService.getAllPendingSightings());
+    return "sightings_map";
```
Comment (Aude11) [inline src/main/java/com/whalespottingjava/controllers/SightingController.java:74]: I would add a line before return for style purpose and clear code

Comment (Aude11) [inline src/main/java/com/whalespottingjava/controllers/SightingController.java:73]: It is more a question but in the future this method would return a sighting objects or array of object sightings?

**Review by Aude11 — COMMENTED** [review]
I have a question regarding the controller

File: src/main/java/com/whalespottingjava/controllers/SightingController.java (hunk 1 above)
Comment (Annie1790) [reply src/main/java/com/whalespottingjava/controllers/SightingController.java:74]: I've added the blank line now

Comment (Aude11) [reply src/main/java/com/whalespottingjava/controllers/SightingController.java:74]: OK, I understand how it works from JAVA/Spring. Should it be display on the admin page then? As it is the admin who wants to see all the pending?

**Review by Aude11 — APPROVED** [review]

**Pull request merged**